docs/content/installation/install_hostdefinition.md

```diff
@@ -0,0 +1,36 @@
+# Installing the `HostDefinition` custom resource
+
+Install the `HostDefinition` custom resource to enable automatic host definitions from within the CSI driver.
+
+Use the following steps to install the `HostDefinition` custom resource, with [GitHub](https://github.com/IBM/ibm-block-csi-operator).
+
+**Note:** Only hosts added after installation are automatically defined with the CSI driver.
+
+1.  Install the operator.
+
+    1. Download the manifest from GitHub.
+
+        ```
+        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.10.0/deploy/installer/generated/host-definition.yaml > host-definition.yaml
+        ```
+
+    2. Verify that the hostdefinition is in the _Ready_ phase.
+
+        ```
+        $> kubectl get hostdefinition
+        NAME                  AGE    PHASE   STORAGE          HOST
+        <k8s_resource_name1>  102m   Ready   <storage_name>   <host_name_w1>
+        <k8s_resource_name2>  102m   Ready   <storage_name>   <host_name_w2>
+        ```
+
+        If in an `Error` state, a retry can be forced using TBD.
+    
+    2.  (Optional) Update the prefix and connectivity fields in the `HostDefiner.yaml`.
```

Comment:
wouldn't it be better to place this step **before** the auto-definitions are complete?

Reply:
changed